miden-lib/build.rs

```diff
@@ -1,10 +1,23 @@
-use assembly::{LibraryNamespace, MaslLibrary, Version};
-use std::{env, fs, fs::File, io, io::BufRead, io::BufReader, io::Write, path::Path};
+use assembly::{
+    ast::{AstSerdeOptions, ProgramAst},
+    LibraryNamespace, MaslLibrary, Version,
+};
+use std::{
+    env, fs,
+    fs::File,
+    io,
+    io::BufRead,
+    io::BufReader,
+    io::Write,
```

Comment:
nit: these can be combined into a single line - i.e., `io::{self, BufRead, BufReader, Write}`.

Reply:
I hoped `cargo fmt` is doing that